learning_cp/graphcoloring/Project.toml

```diff
@@ -3,6 +3,8 @@ BSON = "fbb218c0-5317-5bc6-957e-2ee96dd4b1f0"
 Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
 GeometricFlux = "7e08b658-56d3-11e9-2997-919d5b31e4ea"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
 ReinforcementLearning = "158674fc-8238-5cab-b5ba-03dfc80d1318"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
It seems to me that it was agreed not to put Revise in the .toml because it is development aid. 
@3rdCore ?

Reply:
That's right ! Revise is usefull for development purposes, and it is not required if the user wants simply to execute the code.

Reply:
Okay! I will remove it the next commit then.